### DISCUSSION TITLE
Features

### DISCUSSION THREAD

**Discussion by TeacherRabbit123** [netless-io/flat#1843] [Ideas]
I was wondering if it's possible to put the "Authorization" button like the one in this image? [https://drive.google.com/file/d/1a0uoBUR0PHN5UE-ro6kBYkiPs2Vcy7uz/view?usp=sharing](url)
Also, can we add a reward sticker button? Kids love them. Thanks!

**Comment by TeacherRabbit123** [comment]
It would also be nice if some tools are inaccessible to students. =). The students can close the lesson material. I hope it can be disabled.

**Reply by Leooeloel** [reply]
Here you can turn off the student's operating privileges
![image](https://user-images.githubusercontent.com/29906716/221393377-d8f5a7a1-e70f-4966-b71e-600cdaf7bb25.png)

**Reply by TeacherRabbit123** [reply]
Yes, I saw that. This takes time though as you need to cover the whole screen to select one. Unlike if it there is a shortcut button like that in the picture that I shared.
Thanks
